filebeat/_meta/test/module/foo/multi/config/multi.yml

```
@@ -1,4 +1,4 @@
-type: log
+type: filestream
```

Every Filestream instance must have a unique ID, even for tests and examples it's better to always add it.
```suggestion
type: filestream
id: foo-multi
```